presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -92,6 +92,30 @@
     @Named("databases.presto.jdbc_user")
     private String jdbcUser;
 
+    @Inject(optional = true)
```

Comment:
> Minor: We are moving all of this fields out of this class in a next commit. For the sake of review simplicity couldn't we create a LdapCli tests class at the beginning to avoid moving the code back and forth across the commits?